src/common/components/StepsProgressBar/StepsProgressBar.tsx

```diff
@@ -6,39 +6,45 @@ import checkmarkSVG from 'assets/images/checkmark.svg';
 import styles from './StepsProgressBar.scss';
 
 interface IProps {
-	steps: string[];
+	steps: any;
 	stepsComplete: number;
+	style?: any;
+	icon?: string;
+	titles?: ReactNode[];
 }
 
-const StepsProgressBar: React.FC<IProps> = ({steps, stepsComplete}) => {
+const StepsProgressBar: React.FC<IProps> = ({steps, titles, stepsComplete, icon, style}) => {
+	let barStyles = styles;
+	let barTitles = steps;
+
+	if (style) barStyles = style;
+	if (titles) barTitles = titles;
+
 	const stepClass = (index: number) => ({
-		[styles.step_completed]: stepsComplete > index,
-		[styles.step_inProgress]: stepsComplete === index,
+		[barStyles.step_completed]: stepsComplete > index,
```

Comment:
rewrite with classnames, please. This is not good to always have style objects in components